src/devices/MAX31856/ThermocoupleType.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.MAX31856
+{
+    /// <summary>
+    /// Sets the Thermocouple Type
+    /// </summary>
+    public enum ThermocoupleType : byte
+    {
+        /// <summary>
+        /// Type B thermocouple
```

Comment:
If this is known domain knowledge leave this as is otherwise I'd recommend to write a bit more about what this means

Reply:
This is domain knowledge  (it is a type of thermocouple probe) https://en.wikipedia.org/wiki/Thermocouple#Types